deploy/config/deploy.production.yml

```diff
@@ -11,6 +11,7 @@ servers:
     options:
       # create by ansible
       env-file: '/home/connect/www/commcare-connect/docker.env'
+      m: 1g
```

Comment:
Is `m` an alias for `memory`?

Reply:
yeah, https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory

Reply:
This might need to be `memory` because this is a different tool than docker.

Reply:
https://kamal-deploy.org/docs/configuration#using-accessories-for-database-cache-search-services